app/src/js/common/session.ts

```diff
@@ -11,7 +11,7 @@ import { configureStore } from './configure_store'
 import { Track } from './track'
 
 export const enum ConnectionStatus {
-  SAVED, SAVING, RECONNECTING, UNSAVED
+  JUST_SAVED, SAVED, SAVING, RECONNECTING, UNSAVED
```

Comment:
Can you rename `JUST_SAVED` to `NOTIFY_SAVED`?

Reply:
Changed